README.md

```diff
@@ -507,6 +509,526 @@ source scripts/makeAllPlots.sh figures/ResponseUniformityMaps
 
 this will then execute genericPlotter taking each `*.cfg` file in the `figures/ResponseUniformityMaps` directory.
 
+### 3.b.ii.I Helper Script - Parsing Excel Files to genericPlotter 
+
+- **Main script:** Produce\_Config\_File.py
+- **Additional scripts:** QC3.py, QC4\_HV.py, QC4\_SS.py, QC5\_Gain.py, QC5\_Rate.py
+
+
+**Main script:** This tool is designed to read selected columns from an excel file and produce a config file in the genericPlotter input style described in session: [4.e.iv. Plot Config File](#4eiv-plot-config-file). Following, the produced config file can be used as an input file by genericPlotter to create a graph in the official CMS style.
+
+The following **options** are defined:
+
+Field Name | Type | Description
+---------- | ---- | -----------
+`-h, --help` |  | All the options that are available are displayed.
+`-f, --file` |list of strings  | Add an entry to the input excel file list.
+`--OutputName` |string | Set Output file name, default is: Myconfig.
+`--SelectSheetNum` | int| Select the Excel sheet for reading, default is: 0 (counting starts from 0).
+`--SelectColumnX` |int | Select Column number for X data, default value is 0 (counting starts from 0).
+`--SelectColumnY` | int| Select Column number for Y data, default value is 0 (counting starts from 0).
+`--SelectRowStart` |int |Select the first Row for reading, default is 1 (counting starts from 0).
+`--SelectRowEnd` | int|  Select the last Row for reading, default is 61 (counting starts from 0, the last row is not included).
+`--AxisNDiv` | Two comma separated int's | Defines the number of divisions for a given `TAxis`.  See the `TAxis` [documentation](https://root.cern.ch/doc/master/classTAttAxis.html) for more information.  Between one and three integers can be provided. They are understand as applying to the {X}, {X,Y}, or {X,Y,Z} axes. Default is: 508,510 
+`--CanvDim` | Two comma separated int's | Defines the size of the canvas in pixels.  The first (second) number is for the x (y) direction. Default is: 1000,1000
+`--CanvDrawOpt` | string | Draw option that will be applied to all plots on this canvas. Default is: APE1
+`--CanvGridXY` | Two comma separated bool's | Defines if the grid should be drawn on the canvas.  The first (second) boolean is for the x (y) grid. Default: false,false
+`--LatexLines` | list of strings |Add an entry to the LatexLines list following syntax:'Coord_PadX,Coord_PadY,text'.   
```

Comment:
This is not a list of strings, this is a specific format.  Match with what is shown elsewhere in the `README.md` in [Section 4.e.iv.I](https://github.com/bdorney/CMS_GEM_Analysis_Framework/#4eivi-header-parameters---canvas) ,e.g.:

> Comma separated sequence: float,float,string